### PR TITLE
improved artist about page with carousel similar as review page

### PR DESCRIPTION
**Pull request by jeewanbhusal**
### What this PR does (required):
- This PR contains the improved design of about artist page with carousel effect
- Added recent activity inside artist about page, which contains name of the recent contest in which artist submitted 
- Validated message for the view if artist does not have any personal info and also if no review 

### Screenshots / Videos (required):

https://user-images.githubusercontent.com/84412422/128276228-d249bbb1-77d9-4e7a-b737-4372ac6364df.mov

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: client/src/components/AuthHeader/AuthHeader.tsx [hunk 1]
```diff
@@ -30,7 +30,13 @@ const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
           minWidth="55vw"
           flexWrap="wrap"
           className={classes.linkContainer}
-        >
+        > 
+          <Link to={{ pathname: '/artist', state:'60f101c8d872991344be3628' }}>
```
Comment (tsbolty) [inline client/src/components/AuthHeader/AuthHeader.tsx:34]: Lose the hard coded id here.

File: client/src/pages/AboutArtist/AboutArtist.tsx [hunk 2]
```diff
@@ -60,14 +63,22 @@ export default function AboutArtist(props: { location : { state: string; }}): JS
...
+      if (artistSubmission !== null) {
+        setSubmission(artistSubmission);
+      } else {
+        new Error('Could Not Get Artist Submssions');
+      }
     }
     getInfo();
   }, [id]);
+
+  console.log('submissions are', submission);
```
Comment (tsbolty) [inline client/src/pages/AboutArtist/AboutArtist.tsx:81]: Goodbye to this also please

**Review by tsbolty — COMMENTED** [review]
Very good looking page. I'm wondering if it would be a good idea to maybe click on any avatar of an artist that will take them to their page (we will also have access to that artist's id) rather than a link in the header with the hard-coded id. Let me know what you think.

File: client/src/components/AuthHeader/AuthHeader.tsx (hunk 1 above)
Comment (jeewanbhusal-1) [reply client/src/components/AuthHeader/AuthHeader.tsx:34]: Shit, did i forget that. I thought i removed it :)., I am removing in the next commit. Thank you @tsbolty

File: client/src/pages/AboutArtist/AboutArtist.tsx (hunk 2 above)
Comment (jeewanbhusal-1) [reply client/src/pages/AboutArtist/AboutArtist.tsx:81]: :) yes. thanks

**Comment by jeewanbhusal-1** [comment]
> Very good looking page. I'm wondering if it would be a good idea to maybe click on any avatar of an artist that will take them to their page (we will also have access to that artist's id) rather than a link in the header with the hard-coded id. Let me know what you think.

-- Currently, this artist page is linked through the artist username inside submitted designs page

**Pull request merged**